akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -97,7 +97,7 @@ class Http2ServerDemux extends GraphStage[BidiShape[Http2SubStream, FrameEvent,
         pull(frameIn)
         pull(substreamIn)
 
-        bufferedFrameOut.push(SettingsFrame(Nil)) // server side connection preface
+        bufferedFrameOut.buffer(SettingsFrame(Nil)) // server side connection preface // FIXME don't do this, since we want to GOAWAY right away 
```

Comment:
This is the server-side handshake, so we might change it but if it is just to make h2spec work, I'm not sure I'm for it.